### DISCUSSION TITLE
Why "Run failed"?

### DISCUSSION THREAD

**Discussion by Shunlexxi** [skills/.github#108] [Q&A] [answered]
On following some GitHub skill courses, I kept getting the above-mentioned error, please see attached for more info.

What does it mean and how do I fix it?

![Screenshot_20221107-225734](https://user-images.githubusercontent.com/51166816/200424881-5ad02a76-42f8-4c97-b7c1-1f9b9741cf35.png)
![Screenshot_20221107-225727](https://user-images.githubusercontent.com/51166816/200424907-44c2626a-bafb-43f9-9f96-6c900d5b7c31.png)
![Screenshot_20221107-225658](https://user-images.githubusercontent.com/51166816/200424912-f084b701-6361-4a75-a588-bd31139e6e00.png)

**Comment by sinsukehlab** [comment, marked as answer]
Those errors are not your fault.
Some courses without `get_current_step` jobs in their workflows raise errors on purpose.

**Reply by Shunlexxi** [reply]
Okay! Thank you 
@sinsukehlab

**Reply by sinsukehlab** [reply]
You're welcome.
Thanks for selecting my suggestion as the answer.
